WooCommerce/Classes/Tools/Currency/CurrencyFormatter.swift

```diff
@@ -76,17 +77,20 @@ public class CurrencyFormatter {
     ///     - position: the currency position enum, either right, left, right_space, or left_space.
     ///     - symbol: the currency symbol as a string, to be used with the amount.
     ///
-    func formatCurrency(using stringValue: String, at position: CurrencySettings.CurrencyPosition, with symbol: String, isNegative: Bool) -> String {
+    func formatCurrency(using stringValue: String,
+                        at position: CurrencySettings.CurrencyPosition,
+                        with symbol: String,
+                        isNegative: Bool,
+                        locale: Locale = .current) -> String {
         let space = "\u{00a0}" // unicode equivalent of &nbsp;
         let negative = isNegative ? "-" : ""
 
         // We're relying on the phone's Locale to assist with language direction
-        let current = Locale.current as NSLocale
-        let languageCode = current.object(forKey: NSLocale.Key.languageCode) as? String
+        let languageCode = locale.languageCode
 
         // Remove all occurences of the minus sign from the string amount.
         // We want to position the minus sign manually.
-        let amount = stringValue.replacingOccurrences(of: "-", with: "")
+        let amount = stringValue
```

Comment:
I think we can remove this superfluous assignment, and also the related comment (which seems no more applicable)